handler/content_updated_handler.go

```diff
@@ -48,10 +48,18 @@ func (h *ContentPublished) Handle(ctx context.Context, _ int, msg kafka.Message)
 
 	switch e.DataType {
 	case ZebedeeDataType:
+		if !h.Cfg.EnableZebedeeCallbacks {
+			log.Info(ctx, "zebedee callbacks are disabled, skipping event", log.Data{"data_type": e.DataType})
+			return nil
+		}
```

Comment:
So my question on this is should we even be handling the event if we are just skipping it. This will lead to valid events simply being thrown away. Should they be left for a valid consumer to consume instead? Given these are event handlers, should we not just be skipping consuming particular topics? I'm wondering if the justification of this ticket needs reconsidering? 🤔 Alternatively should we just return the event as unhandled? 🤔